odalic/src/main/java/cz/cuni/mff/xrg/odalic/outputs/rdfexport/DefaultRDFExporter.java

```diff
@@ -0,0 +1,32 @@
+package cz.cuni.mff.xrg.odalic.outputs.rdfexport;
+
+import java.io.StringWriter;
+
+import org.eclipse.rdf4j.model.Model;
+import org.eclipse.rdf4j.rio.RDFFormat;
+import org.eclipse.rdf4j.rio.Rio;
+
+/**
+ * Default implementation of the {@link RDFExporter}.
+ * 
+ * @author Josef Janoušek
+ *
+ */
+public class DefaultRDFExporter implements RDFExporter {
+
+  /**
+   * The default export implementation. 
+   * 
+   * @see cz.cuni.mff.xrg.odalic.outputs.rdfexport.RDFExporter#export(org.eclipse.rdf4j.model.Model, org.eclipse.rdf4j.rio.RDFFormat)
+   */
+  @Override
+  public String export(Model rdfModel, RDFFormat rdfFormat) {
```

Comment:
There should be also alternative method, which export the Model directly to a File (not writing to a String first and then writing to a file)